v4l2-request-test.c

```diff
@@ -217,6 +217,7 @@ static void setup_config(struct config *config)
 	config->drm_driver = strdup("sun4i-drm");
 
 	config->preset_name = strdup("bbb-mpeg2");
+	asprintf(&config->slices_path, "data/%s", config->preset_name);
```

Comment:
I'd rather keep this in main() so that we alloc once instead of alloc + free + alloc when user specifies another path.

Reply:
ok. [v5] update already send.